.werft/installer-tests.ts

```diff
@@ -387,12 +387,12 @@ function runIntegrationTests() {
     werft.done("run-integration-tests");
 }
 
-function callMakeTargets(phase: string, description: string, makeTarget: string, failable: boolean = false) {
+async function callMakeTargets(phase: string, description: string, makeTarget: string, failable: boolean = false) {
```

Comment:
Can we add an `await` in the [call to this fn ](https://github.com/gitpod-io/gitpod/pull/13627/files#diff-073c7685dea478fdf0c74dd43a01d49232bd3b941d8dac00658e9a885bbfb1e8R471) in `cleanup` function. Currently since it is being called async, we are getting weird behavior https://werft.gitpod-dev.com/job/gitpod-custom-nvn-test-ad-stream.1. The call to list the remaining state gets triggered which is not empty since deletion is still going on. And this sends quite a lot of alerts to slack.